opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingFilter.java

```diff
@@ -59,7 +57,8 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 spanBuilder.asChildOf(extractedSpanContext);
             }
 
-            Span span = spanBuilder.start();
+            Span span = spanBuilder.startManual();
+            tracer.makeActive(span);
```

Comment:
`tracer.makeActive(span);` - will make span active forever? (if thread is taken from thread pool and returned back when execution is finished)